authenticators.md

```diff
@@ -27,4 +27,28 @@ The default set of functions that we want to provide are:
 We are evaluating the possibility of "smart" request authenticators, that can
 take any request and delegate to the specific request authenticator.  For
 example, a "OAuthRequestAuthenticator", which will delegate to the proper
-OAuth request authenticator.
\ No newline at end of file
+OAuth request authenticator.
+
+##Types of Authentication
+
+There are multiple ways that clients can present their authentication credentials to the web framework: 
+
+**Cookie Authentication**
+
+The framework integration should have the ability to accept an OAuth access or refresh token in the cookies as detailed in the [cookie authentication](cookie-authentication.md) document.
+
+**Bearer Authentication**
+
+The framework integration should be able to read an access token presented in a Bearer Authentication scheme. As an example, this is a HTTP header that looks like this:
+
+```
+Authorization: Bearer <OAuth2AccessToken>
+```
+
+**Basic Authentication**
+
+The framework integration should be able to read API keys presented in Basic Authentication form. As an example, this is a HTTP header that looks like this:
```

Comment:
The implementation of our client credential workflow requires API Keys are that are a resource of an account, and that account must be enabled and accessible by the application the was used during the issuing of the token.